translations/units.tex/pl.po

```diff
@@ -124,6 +124,21 @@ msgid ""
 "\\textbf{Reinforcing} requires that your Town has a Citadel in addition to a Dwelling of that Unit's Level.\\par\n"
 "\n"
 msgstr ""
+"\\hypertarget{Units}{In}\\index{Units} Oprócz Bohatera i powiązanej z nim talii, gracze posiadają także \\textbf{talię Kart Jednostek}, która reprezentuje armię poruszającą się z ich Bohaterami.\n"
```

Comment:
Tutaj In jest częścią tekstu "In addition", więc trzeba będzie to czymś zastąpić

Reply:
Nie wystarczy wywalić {In} ?

Reply:
Taka zmiana
\\hypertarget{Jednostki}{Wraz}\\index{Jednostki} z Bohaterem i powiązaną z nim Talią,